cmd/krew/cmd/list.go

```diff
@@ -51,7 +51,7 @@ Remarks:
 					names = append(names, name)
 				}
 				sort.Strings(names)
-				fmt.Fprintln(os.Stdout, strings.Join(names, "\n"))
+				fmt.Fprint(os.Stdout, strings.Join(names, "\n"))
```

Comment:
This command's output should end with a `\n`.  so I suspect fmt.Println was correct here.

Reply:
see my comment  for  lines()

Reply:
If that is a requirement, you are right. `Fprintln` only has the disadvantage that an empty list gets printed as "\n\n" instead of "\n" or "".

I'll think of a better solution then.

Reply:
Hmm I don't see \n\n.

```
export KREW_ROOT=$(mktemp -d)
kubectl krew update
kubectl krew list
```

this command prints:

```
PLUGIN VERSION\n
```

when you redirect it  to a non-TTY stdout, it prints just `\n` which I think is ok?

Reply:
Hmm.. true. I don't remember how I encountered the double "\n\n" :/